kotlinx-coroutines-core/src/main/kotlin/kotlinx/coroutines/experimental/CoroutineExceptionHandler.kt

```diff
@@ -51,7 +52,17 @@ public interface CoroutineExceptionHandler : CoroutineContext.Element {
     /**
      * Key for [CoroutineExceptionHandler] instance in the coroutine context.
      */
-    companion object Key : CoroutineContext.Key<CoroutineExceptionHandler>
+    companion object Key : CoroutineContext.Key<CoroutineExceptionHandler> {
+        /**
+         * Creates new [CoroutineExceptionHandler] instance.
+         * @param handler a function which handles exception thrown by a coroutine
+         */
+        public operator fun invoke(handler: (CoroutineContext, Throwable) -> Unit): CoroutineExceptionHandler =
+            object: AbstractCoroutineContextElement(Key), CoroutineExceptionHandler {
```

Comment:
Potentially on every `launch`. It is not unthinkable to start a coroutine on every frame in a UI application.